app/controllers/sessions_controller.rb

```diff
@@ -8,6 +8,7 @@ def new
   def create
     @user = User.find_by(email: params[:user][:email])
     if @user && @user.authenticate(params[:user][:password])
+
```

Comment:
Please remove this extra new-line. It doesn't seem like this change is actually related to the bugfix, so we want to make sure the git history changes for the fewest number of files possible.